src/IdeaStatiCa.BimImporter/Geometry.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace IdeaStatiCa.BimImporter
 {
-	internal class Geometry : IGeometry
+	public class Geometry : IGeometry
 	{
 		private static readonly IIdeaLogger _logger = IdeaDiagnostics.GetLogger("ideastatica.bimimporter.geometry");
```

Comment:
assembly should not depend on IdeaStatiCa.Diagnostics - use its wrapper IPluginLogger https://github.com/idea-statica/ideastatica-public/blob/main/src/IdeaStatiCa.Plugin/IPluginLogger.cs